app/src/main/java/com/jerboa/Utils.kt

```diff
@@ -379,7 +379,49 @@ fun LazyListState.isScrolledToEnd(): Boolean {
     return out
 }
 
+/*
+ * Parses a "url" and returns a spec-compliant Url:
+ *
+ * - https://host/path - leave as-is
+ * - /c/community -> https://currentInstance/c/community
+ * - /c/community@instance -> https://instance/c/community
+ * - !community@instance -> https://instance/c/community
+ * - @user@instance -> https://instance/u/user
+ */
+fun parseUrl(url: String): String? {
+    if (url.startsWith("https://")) {
```

Comment:
Can we also handle http (not https) links?

Reply:
Done. :)